modules/javafx.graphics/src/test/java/test/javafx/scene/NodeTest.java

```diff
@@ -2023,4 +2021,36 @@ public interface StubRectAccessor {
         }
 
     }
+
+    @Test
+    public void disabledFlagUpdatesChildrenDisabledFlag() {
+        var g = new Group();
+        var n1 = new Rectangle();
+        var n2 = new Rectangle();
+        g.getChildren().addAll(n1, n2);
+        assertFalse(g.isDisabled());
+        assertFalse(n2.isDisabled());
+        assertFalse(n2.isDisabled());
+
+        g.setDisable(true);
+        assertTrue(g.isDisabled());
+        assertTrue(n2.isDisabled());
+        assertTrue(n2.isDisabled());
+    }
```

Comment:
I suggest to continue this test with a `g.setDisable(false)` to check both directions. The first group of assertions test the initial state, not a change to `false`.

Same for the other test.

Reply:
Done.